apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -1,20 +1,224 @@
+      'Select a Rating',
+      '1 - Unusable', 
+      '2 - Needs Rework', 
+      '3 - Acceptable',
+      '4 - Exceeds Expectations',
+      '5 - Excellent',
+    ]
+
+    return(
+      <div>
+        <IssueTitle>{issue.title}</IssueTitle>
+        
+        <SafeLink
+          href={issue.url}
+          target="_blank"
+          style={{ textDecoration: 'none', color: '#21AAE7' }}
+        >
+          <IssueLinkRow>
+            <IconGitHub color="#21AAE7" width='14px' height='14px' />
+            <Text style={{ marginLeft: '6px'}}>{issue.repo} #{issue.number}</Text>
+          </IssueLinkRow>
+        </SafeLink>
+
+        <SubmissionDetails>
+          <UserLink>
+            <img src={submittor.avatar} style={{ width: '32px', height: '32px', marginRight: '10px'}} />
+            <SafeLink
+              href={submittor.url}
+              target="_blank"
+              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
+            >
+              {submittor.name ? submittor.name : submittor.login}
+            </SafeLink>
+            applied {submissionDateDistance} ago
+          </UserLink>
 
-const ReviewWork = ({ issues, rate, onSubmit }) => <div>Review Work</div>
-
-ReviewWork.propTypes = {
-  /** array of issues to allocate bounties on */
-  issues: PropTypes.arrayOf(
-    PropTypes.shape({
-      id,
-      level,
-    })
-  ),
-  /** base rate in pennies */
-  rate: PropTypes.number,
-  /** callback fn */
-  onSubmit: PropTypes.func,
+          <Separator/>
+
+          <FieldTitle>Proof of Work</FieldTitle>
+          <SafeLink
+            href={work.proof}
+            target="_blank"
+            style={{ textDecoration: 'none', color: '#21AAE7' }}
+          >
+            <DetailText>{work.proof}</DetailText>
+          </SafeLink>
+
+          <CommentsCheck comments={work.comments} />
+          <DetailText>{work.comments}</DetailText>
+
+          <FieldTitle>Hours Worked</FieldTitle>
+          <DetailText>{work.hours}</DetailText>
+
+        </SubmissionDetails>
+
+        <FormField 
+          label="Quality Rating" 
+          required
+          input={
+            <DropDown
+              items={ratings}
+              onChange={this.generateRatingChange()}
+              active={this.state.rating}
+            />
+          }
+        />
+
+        <FormField
+          label="Feedback"
+          input={
+            <TextInput.Multiline
+              name='feedback'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
+              onChange={this.changeField}
+              placeholder="Do you have any feedback to provide the contributor?"
+              wide
+            />
+          }
+        />
+
+        <ReviewRow>
+          <ReviewButton emphasis="negative" onClick={this.onReviewApplicationReject}>Reject</ReviewButton>
+          <ReviewButton emphasis="positive" onClick={this.onReviewApplicationAccept}>Accept</ReviewButton>
```

Comment:
These buttons need a little horizontal separation IMHO

Reply:
Changed the styled button width from a raw pixel value to a percentage, so they should be separated in any view context